zh_CN/LC_MESSAGES/django.po

```diff
@@ -7757,9 +7757,10 @@ msgid ""
 "                This value may contain only letters, numbers, and @/./+/-/_ characters.\n"
 "                "
 msgstr ""
+"\n"
 "请注意用户名将公开可见。\n"
 "这是必填字段，不能与他人重复，且须不超过 150 个字符。\n"
-"字段内容仅可包含拉丁字母、数字、@ . + - _ 等符号。"
+"字段内容仅可包含拉丁字母、数字、@ . + - _ 等符号."
```

Comment:
We need to change two things here for the `msgstr`:
- prefix each of the three textual lines with same number of spaces as the `msgid` lines have
- add a final line of spaces to match as well

So overall something like:
```
msgstr ""
"\n"
"                请注意用户名将公开可见。\n"
"                这是必填字段，不能与他人重复，且须不超过 150 个字符。\n"
"                字段内容仅可包含拉丁字母、数字、@ . + - _ 等符号。\n"
"                "
```